src/Makefile.am

```diff
@@ -33,11 +33,11 @@ rar2fs_SOURCES = 	dllext.cpp \
 			dllext.hpp
 
 rar2fs_LDADD = $(PTHREAD_CFLAGS) $(LDFLAGS) $(UNRAR_LDFLAGS) $(FUSE_LDFLAGS) \
-		$(PTHREAD_LIBS) $(FUSE_LIBS) $(UNRAR_LIBS) $(LIBICONV) -lstdc++
-rar2fs_LINK = $(CC) -o $@
+		$(PTHREAD_LIBS) $(FUSE_LIBS) $(UNRAR_LIBS) $(LIBICONV)
+rar2fs_LINK = $(CXX) -o $@
 mkr2i_SOURCES = mkr2i.c
 mkr2i_LDADD = $(LDFLAGS)
-mkr2i_LINK = $(CC) -o $@
+mkr2i_LINK = $(CXX) -o $@
```

Comment:
This does not look right? The mkr2i is a pure C application, it does not use the same linker flags as rar2fs and does not require libstdc++.